calculateFrequency.js

```diff
@@ -0,0 +1,14 @@
+function calculateFrequency(string) {
+    let toReturn = {};
+    for (let i = 0; i < string.length; i++) {
+        if (string.charCodeAt(i) > 96 && string.charCodeAt(i) < 123) {
+            toReturn[string.charAt(i)] = 0;
+        }
+    }
+    for (let i = 0; i < string.length; i++) {
+        if (string.charCodeAt(i) > 96 && string.charCodeAt(i) < 123) {
+            toReturn[string.charAt(i)] = toReturn[string.charAt(i)] + 1;
+        }
+    }
+    return toReturn;
+}
```

Comment:
EOF should be a blank line always. WHY?? Your job is to find out why and let us know.